focus-group.js

```diff
@@ -109,3 +107,36 @@ export function menuKeyUX(options) {
     }
   }
 }
+
+export function findGroupNodeByEventTarget(eventTarget) {
+  let supportedRoles = {
+    'menuitem': ["menu", "menubar"],
+    'option': ["listbox"],
+    'tab': ["tablist"]
+  }
+  let itemRole = eventTarget.role
+  let groupRoles = supportedRoles[itemRole]
+  if (!groupRoles) return null
+
+  let result
+  groupRoles.forEach((role) => {
```

Comment:
It is better to use `for of` here and `return` on found. Because otherwise you will do unnecessary search for `menubar` if `menu` was already found.